circleci/images/pgupgradetester/files/sbin/install-testdeps

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# make bash behave
+set -euxo pipefail
+IFS=$'\n\t'
+
+# this script needs root to install packages
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+OLD_PG_VERSION=10
+NEW_PG_VERSION=11
+
+# explicitly set user/group IDs
+groupadd -r circleci --gid=1729
+useradd -mg circleci --uid=1729 --shell=/bin/bash circleci
+
+echo "Running apt-get update..." >&2
+apt-get update
+apt-get install -y --no-install-recommends \
+  apt-transport-https \
+  ca-certificates \
+  curl \
+  debian-archive-keyring
+
+pkg_cloud_url="https://packagecloud.io/install/repositories/citusdata/the-process/script.deb.sh"
```

Comment:
I havent found any usage of it, therefore I am removing it.

Reply:
Well as far as I understand this package contains a smaller `postgresql-server-dev` package. So we install our repository and pin its priority to the highest so that when `apt-get install postgresql-server-dev-..` is called, that is installed from our repository.